cmd/neofs-adm/internal/modules/morph/initialize_deploy.go

```diff
@@ -147,27 +152,137 @@ func (c *initializeContext) deployContracts(method string) error {
 	}
 	nnsHash := nnsCs.Hash
 
+	totalGasCost := int64(0)
+	w := io2.NewBufBinWriter()
+
 	var keysParam []smartcontract.Parameter
 
+	// Update script size for a single-node committee is close to the maximum allowed size of 65535.
```

Comment:
If update script size is close to tx limit, maybe we can split alphabet contract deployment and other contract deployment?

Reply:
It is now 48k which is 17k less than 65k. I think we can safely do everything in one transaction unless we add another contract.